cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/strategy/PrioritizeSmallMovementExecutionTaskStrategy.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2018 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.executor.strategy;
+
+import com.linkedin.kafka.cruisecontrol.executor.ExecutionTask;
+
+public class PrioritizeSmallMovementExecutionTaskStrategy extends AbstractExecutionTaskStrategy {
+
+  @Override
+  public int compareTask(ExecutionTask task1, ExecutionTask task2) {
```

Comment:
I am wondering whether we can support the use of multiple strategies at once. For example, a user might prioritize movement first based on `PostponeUrpExecutionTaskStrategy`, then based on `PrioritizeSmallMovementExecutionTaskStrategy`. We can support this by chaining requested comparators.